presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -77,6 +77,13 @@ String Functions
 
     Removes leading whitespace from ``string``.
 
+.. function:: ltrim(string, chars) -> varchar
+
+    Removes the longest substring containing only characters in ``chars`` from the beginning of the ``string``.
```

Comment:
```suggestion
    Removes the longest substring containing only characters in ``chars`` from the beginning of the ``string``. ::
```

Reply:
Got it. split_to_map was also broken. Fixed that too.